roles/pbench_storage/vars/main.yml

```diff
@@ -5,3 +5,7 @@ pbench_storage_device: '/dev/vda'
 pbench_storage_device_partition: 3
 # Pbench agent default directory.
 pbench_storage_mount_path: '/var/lib/pbench-agent'
+# Pbench results redirector host.
+pbench_results_redirector: 'pbench.perf.lab.eng.bos.redhat.com'
```

Comment:
Could this be read from an environment variable?

Reply:
It could if we need it.